### ISSUE TITLE
NVDA and VO read line modes not supported 

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#14104]
### Description

The various NVDA reading modes are yet to be consistently supported.

NVDA will currently read out everything the app when the Read All option is activated. However, the experiencing of reading out all the individual components on the screen does not appear to be fully supported.

NVDA screen reader users anticipate clicking on up and down arrow keys, as well as utilizing the Read Line option to read lines from the app, regardless of whether or not those lines/controls receive focus. 

I've noticed similar behavior on Mac w/ VoiceOver VO+L read line shortcut.

### Steps to Reproduce

1. Launch File > New > .NET MAUI app
2. Launch NVDA
3. Use arrow keys and/or Read Line option to navigate app

1. Launch File > New > .NET MAUI app
2. Launch VoiceOver
3. VO+L to read line

### Link to public reproduction project repository

dotnet/maui

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows

**Comment by rachelkang** [comment]
Noticing something similar with VoiceOver on Mac
VO+L doesn't read the current line.

**Comment by Zhanglirong-Winnie** [comment]
**Verified this issue is not repro using Visual Studio Enterprise 17.6.5 (Build 417). The line content can be read using VoiceOver and the arrow keys.**
